src/index.html

```diff
@@ -21,7 +24,396 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+      <nav class="navi-bar">
+        <ul class="navi-menu">
+          <li class="navi-item">
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="#"
+              class="navi-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="#"
+              class="navi-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="#"
+              class="navi-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="#"
+              class="navi-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="#"
+              class="navi-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="#"
+              class="navi-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="#"
+              class="navi-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="card-container">
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          class="product-card__image"
+          alt="imac"
+        />
+
+        <div class="product-card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product-card__code">Product code: 195434</div>
+
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star stars__star--inactive"></div>
+
+          <span class="reviews">Reviews: 5</span>
+        </div>
+
+        <div class="price">
+          <span class="price__text">Price:</span>
+          <span class="price__number">$2,199</span>
+        </div>
+
+        <a
+          class="button-buy"
+          href="#"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
```

Comment:
It seems like you have a lot of repeated code for each card. Consider creating a separate component for the card and reusing it. This can help make your code more DRY (Don't Repeat Yourself), easier to maintain, and less prone to errors.